src/Pkg.jl

```diff
@@ -186,6 +186,7 @@ const update = API.up
   - `coverage::Bool=false`: enable or disable generation of coverage statistics.
   - `julia_args::Union{Cmd, Vector{String}}`: options to be passed the test process.
   - `test_args::Union{Cmd, Vector{String}}`: test arguments (`ARGS`) available in the test process.
+  - `allow_autoprecomp::Bool=true`: allow auto-precompilation to be done in test environment, if not disabled globally
```

Comment:
LGTM, but is this necessary given that you can use the ENV variable? Maybe doesn't matter.

Reply:
Yeah, fair point. Removed